index.js

```diff
@@ -149,6 +150,26 @@ app.post("/user_signup", async (req, res) => {
   }
 });
 
+app.post("/user_auth", async (req, res) => {
+  var reqBody = req.body;
+  var user_exists = await get_auth_user_info_by_email(reqBody.email);
+  if (!user_exists) {
+    return res
+      .status(400)
+      .json({ error: "user doesn't exist, please signup!" });
+  }
+  if (await bcrypt.compare(reqBody.password, user_exists["password"])) {
+    res.status(200).json({ success: "user authenticated!" });
+    var accessToken = await jwt.sign(
+      user_exists,
```

Comment:
user object has confidentail information like password, created_at etc. Don't put entire object, only put the details required in the jwt